SPCTR-2361-Post-timeline in style setting the left padding side is not changing in front end (#2753)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Display conditions - Undefined `key/index` when none is set for Operating system.
* Fix: Forms/Tabs/Table of contents - JS throws an error in console on blog/home page.
* Fix: Team - On responsive devices preset images not working as expected.

> * Fix: Post-timeline - Left Padding not applicable on the frontend.

Reviewer: ```suggestion
* Fix: Post Timeline - Left Padding not applicable on the frontend.
```

> * Fix: Post-timeline - in style setting the left padding side is not changing in front end

Reviewer: ```suggestion
* Fix: Post-timeline - Left Padding not applicable on the frontend.
```
